data/recipes/gcp_logging_cloudsql_ts.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "gcp_logging_cloudsql_ts",
+  "description": "Collects GCP logs from Cloud SQL instances for a project and exports them to Timesketch.",
+  "short_description": "Collects GCP logs from Cloud SQL instances for a project and exports them to Timesketch.",
+  "modules": [
+    {
+      "wants": [],
+      "name": "GCPLogsCollector",
+      "args": {
+        "project_name": "@project_name",
+        "filter_expression": "logName:\"projects/@project_name/logs/cloudsql.googleapis.com\" timestamp>\"@start_date\" timestamp<\"@end_date\""
+      }
+    },
+    {
+      "wants": [
+        "GCPLogsCollector"
+      ],
+      "name": "GCPLoggingTimesketch",
+      "args": {}
+    },
+    {
+      "wants": [
+        "GCPLoggingTimesketch"
+      ],
+      "name": "TimesketchExporter",
+      "args": {
+        "incident_id": "@incident_id",
+        "token_password": "@token_password",
+        "sketch_id": "@sketch_id"
+      }
+    }
+  ],
+  "args": [
+    [
+      "project_name",
+      "Name of GCP project to collect logs from",
+      null
+    ],
```

Comment:
Could you have these on one line to be consistent with how the other recipes are formatted?

Reply:
Done